reports/Oregon.md

```diff
@@ -1,5 +1,41 @@
 ## Eugene
 
+### Peaceful protestor tear gassed | May 30th
+
+Footage shows a small cluster of protestors chanting "I can't breathe." At 1:15, without warning, police fire tear gas. Most protestors flee. Some canisters can be seen thrown back towards police.
+
+Police then proceed down the street. More tear gas is fired at 4:30.
+
+tags: tear-gas, protestor
+
+id: or-eugene-2
+
+**Links**
+
+* https://www.youtube.com/watch?v=dPTr54DdTdY
+* https://www.reddit.com/r/Eugene/comments/gtbsvq/eugene_police_teargasses_downtown/
+
+
+### Reporter shot with tear gas canister | May 31st
+
+Footage shows police giving curfew dispersal order to protestors. At one point, the filmer and his fellow journalist identify themselves as a reporter. One officer says it doesn't matter, they need to leave. They continue to monitor the protest and are later joined by a reporter for the Eugene Weekly.
```

Comment:
Is "it doesn't matter, they need to leave" a quote?

Reply:
It's a pseudo-quote I'll make it clearly not one